src/sentry/api_gateway/proxy.py

```diff
@@ -61,15 +61,14 @@ def proxy_request(request: HttpRequest, org_slug: str) -> StreamingHttpResponse:
     header_dict = clean_proxy_headers(request.headers)
     # TODO: use requests session for connection pooling capabilities
     assert request.method is not None
-    query_params = getattr(request, request.method, None)
+    query_params = request.GET
     try:
         resp = external_request(
             request.method,
             url=target_url,
             headers=header_dict,
             params=dict(query_params) if query_params is not None else None,
-            files=getattr(request, "FILES", None),
-            data=getattr(request, "body", None) if not getattr(request, "FILES", None) else None,
+            data=request,
```

Comment:
The request object is a file like object and will stream its body

Reply:
Do you not need to stream files? We have a few endpoints that take file uploads (avatars, debug symbols etc).

Reply:
Yup.  And they will receive those files and do just as they did.  The point is, if you're proxying the request, you don't need to do anything.

Files, by the way, are just encoded in the body.  The request body always contains everything if you don't buffer and parse it.